docs/application-connector/docs/02-03-proxy-service.md

```diff
@@ -7,18 +7,30 @@ The Application Gateway sends the requests from Lambda functions and services in
 
 >**NOTE:** The system creates an Access Service for every external API registered by the Application Registry.
 
-The following diagram illustrates how the Application Gateway interacts with other components and external APIs secured with OAuth and protected against cross-site request forgery (CSRF) attacks.
+The following diagram illustrates how the Application Gateway interacts with other components and external APIs
+which might be unsecured or secured with various security mechanism and also protected against cross-site request forgery (CSRF) attacks.
 
 ![Application Gateway Diagram](./assets/003-architecture-proxy-service.svg)
 
 1. A lambda function calls the Access Service. The name of every Access Service follows this format: `app-{application-name}-{service-id}`
 2. The Access Service exposes the Application Gateway.
-3. The Application Gateway extracts the Application name and the service ID from the name of the Access Service name. Using the extracted Application name, the Application Gateway finds the respective Application custom resource and obtains the information about the registered external API, such as the API URL and the OAuth server URL.
+3. The Application Gateway extracts the Application name and the service ID from the name of the Access Service name. Using the extracted Application name, the Application Gateway finds the respective Application custom resource and obtains the information about the registered external API, such as the API URL and security credentials.
 4. The Application Gateway gets a token from the OAuth server.
-5. The Application Gateway gets a CSRF token from the endpoint exposed by the upstream service.
-6. The Application Gateway calls the target API using the OAuth and CSRF tokens.
+5. The Application Gateway gets a CSRF token from the endpoint exposed by the upstream service. (optional step based on API registration in App Registry)
```

Comment:
```suggestion
5. The Application Gateway gets a CSRF token from the endpoint exposed by the upstream service. This step is optional and is valid only for the API registration in the Application Registry.
```

Reply:
It is not exactly what I was trying to express:
>This step is optional and is valid only for the API which was registered with CSRF token turned on.
Something like that is more accurate IMO.